vm/hv1/hvdef/src/lib.rs

```diff
@@ -3611,7 +3611,7 @@ impl MessagePayload for HvX64VmgexitInterceptMessage {}
 pub struct HvRegisterVpAssistPage {
     pub enabled: bool,
     #[bits(11)]
-    _reserved: u64,
+    pub reserved: u64,
```

Comment:
No. We have discussed this before. If someone defines some of these bits later, then this breaks any code using this field to check for unsupported bits--suddenly, the code "supports" the new field automatically (but most likely just ignores it).

The existing pattern is the right one.